pkg/sql/parsers/dialect/mysql/mysql_sql.y

```diff
@@ -2052,6 +2057,147 @@ into_table_name:
         $$ = $1
     }
 
+export_data_stmt:
+    select_no_parens export_data_param
```

Comment:
The construction of the whole grammar should be in simple_select, refer to tidb or MySQL for details